gulp/tasks/browserSync.js

```diff
@@ -1,7 +1,11 @@
-var gulp            = require('gulp'),
-    browserSync     = require('browser-sync'),
-    config          = require('../config').browserSync;
+var gulp = require('gulp'),
+    browserSync = require('browser-sync'),
+    config = require('../config').browserSync;
 
-gulp.task('browserSync', ['jekyll-build'], function() {
-    browserSync(config);
-});
+(function() {
```

Comment:
Yep, you're right. IIFEs don't do anything extra in node as each module has its own scope.

I added the IIFEs to make eslint's `strict` rule happy (`'use strict';` must be invoked at function scope) but it looks safe to turn that rule off for the `/gulp` directory as the files in it are commonjs modules.